.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result give me those PR assignees sorted alphabetically in a separate sentence.

Reply:
Why are the assignees not sorted alphabetically?

Reply:
What do I say to make sure it's sorted correctly next time?

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result give me those PR assignees in a separate sentence, this output should be sorted alphabetically using the jq command and verify that it is sortes alphabetically.

Reply:
The assignees are not sorted alphabetically. What do I need to say to make sure that they do next time?

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result give me those PR assignees sorted alphabetically in a separate sentence.

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number & title.

From the output give me those PR assignees sorted alphabetically in a separate sentence. Make sure they are sorted alphabetically.

Reply:
Sort the assignee two times

Reply:
Why is jjw24 not before VictoriousRaptor?

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number and the title.

From the result give me those PR assignees in a separate sentence, sorted in case insensitive alphabetical order.

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result give me those PR assignees in a separate sentence, sorted in case insensitive alphabetical order and exclude jjw24.

Reply:
Try again

Reply:
Try agy

Reply:
Try again

Reply:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

From the result give me those PR assignees in a separate sentence, sorted in case insensitive alphabetical order.